src/cocaine-app/sched/ttl_cleanup_starter.py

```diff
@@ -0,0 +1,119 @@
+import logging
+
+from infrastructure import infrastructure
+import jobs
+from mastermind_core.config import config
+import storage
+import time
+import datetime
+from yt_worker import YqlWrapper
+
+
+logger = logging.getLogger('mm.sched.ttl_cleanup')
+
+class TtlCleanupStarter(object):
+    def __init__(self, scheduler):
+        scheduler.register_periodic_func(self._do_ttl_cleanup,
+                                         period_default=60*15,  # 15 minutes
```

Comment:
fixed in https://github.com/yandex/mastermind/pull/434/commits/968d0fbf32e9a25f5713cb1d2e5c4d336cf6a401